tests/features/asset_distribution/add_asset_distribution.feature

```diff
@@ -5,22 +5,19 @@ Feature: Add asset distribution through the UI
   I need to be able to add "Asset distribution" RDF entities through the UI.
 
   Scenario: "Add distribution" button should only be shown to moderators.
-    Given the following collection:
-      | title | Asset Distribution Test |
-      | logo  | logo.png                |
-    And the following solution:
+    Given the following solution:
       | title       | Asset random name                |
       | description | Some reusable random description |
-      | collection  | Asset Distribution Test          |
+    And the following collection:
+      | title      | Asset Distribution Test |
+      | logo       | logo.png                |
+      | affiliates | Asset random name       |
     And the following asset release:
       | title         | Asset release random name        |
       | description   | Some reusable random description |
       | is version of | Asset random name                |
-    When I am logged in as a "moderator"
+    When I am logged in as a "facilitator" of the "Asset random name" solution
     And I go to the homepage of the "Asset release random name" asset release
-
-    # Click the + button.
-    Then I click "Add"
     Then I should see the link "Add distribution"
```

Comment:
Isn't it necessary to click the "Add" link in normal use? Remember that these scenarios are intended to describe the user behaviour. The user doesn't see the link "Add distribution" on the screen at this point, because it is hidden behind the plus button.